model/Comment/DacMentionEligibleUsersProvider.php

```
@@ -0,0 +1,110 @@
+use core_kernel_classes_Resource;
+use oat\generis\model\data\Ontology;
+use oat\generis\model\GenerisRdf;
+use oat\taoDacSimple\model\PermissionProvider;
+use oat\taoDacSimple\model\RolePrivilegeRetriever;
+use oat\taoItems\model\Comment\MentionEligibleUsersProviderInterface;
+use tao_models_classes_UserService;
+
+/**
+ * Restricts mention candidates to users with effective READ on the resource (DAC).
+ */
+final class DacMentionEligibleUsersProvider implements MentionEligibleUsersProviderInterface
```

PHPStan (2.2.8)

[error] Class oat\taoDacSimple\model\Comment\DacMentionEligibleUsersProvider implements unknown interface oat\taoItems\model\Comment\MentionEligibleUsersProviderInterface.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(interface.notFound)